docs/getting-started-guides/windows/README.md

```diff
@@ -0,0 +1,163 @@
+# Running Windows Server Containers using Kubernetes
+Microsoft, in collaboration with Docker, is introducing a new feature called Windows Server Containers with Windows Server 2016, which enables Docker containers to run on Windows. We have enhanced Kubernetes to support Windows Server Containers (and Windows Server 2016 as a container host node).
+
+Windows Server and .NET still account for a significant portion of the workloads in the enterprise, and with this work, customers will be able to run Windows-based and .NET-based applications inside containers on Kubernetes, making Kubernetes the first fully functional cross-platform cluster manager. Services that span Windows-based and Linux-based containers are now a reality.
+
+Windows Server Containers will be supported on Kubernetes as an alpha feature. Kubernetes control plane (API Server, Scheduler, Controller Manager etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server.
+
+## Prerequisites
+With the alpha release, we support Windows Server Containers for Kubernetes using the following:
+
+1. Kubernetes control plane running on existing Linux infrastructure (version 1.5 or later)
+2. Kubenet network plugin setup on the Linux nodes
+3. Windows Server 2016 (RTM version 10.0.14393 or later)
+4. Docker Version 1.12.2-cs2-ws-beta or later
+
+## Networking
+Network is achieved using L3 routing. Because third-party networking plugins (e.g. flannel, calico, etc) don’t natively work on Windows Server, we relied on existing technology that is built into the Windows and Linux operating systems. In this L3 networking approach, we chose a /16 subnet for the cluster nodes, and we assign a /24 subnet to each worker node. All pods on a given worker node will be connected to the /24 subnet. This allows pods on the same node to communicate with each other. In order to enable networking between pods running on different nodes, routing features that are built into Windows Server 2016 and Linux are used.
+
+### Linux
+The above networking approach is already supported on Linux using a bridge interface, which essentially creates a private network local to the node. Similar to the Windows side, routes to all other pod CIDRs must be created in order to send packets via the “public” NIC.
+
+### Windows
+To support networking requirements, we will need the following configuration on each Kubernetes Windows Server node:
```

Comment:
Replace "To support networking requirements, we will need the following configuration on each Kubernetes Windows Server node" with "Each Window Server node should have the following configuration".